packages/pubsub/src/index.js

```diff
@@ -291,6 +292,7 @@ PubSub.prototype.determineBaseUrl_ = function() {
 
   this.options.servicePath = baseUrlParts[0];
   this.options.port = baseUrlParts[1];
+  this.options.sslCreds = grpc.credentials.createInsecure()
```

Comment:
Just to beat the linting task to it, semi-colon missing!

Reply:
Take that, linting task!

Reply:
Doh